packages/builder/setup.js

```diff
@@ -1,14 +1,14 @@
-const cypressConfig = require("../cypress.json")
```

Comment:
I would possibly rename this file as `testSetup.js` or similar as it's only for testing.

Reply:
Similar to other comment, we can move this directly into qa-core now